schema_utils.py

```diff
@@ -265,3 +265,12 @@ def is_regex(obj):
     def matches_regex(obj, regex):
         """Ensures that `obj` (a string) matches the given regex."""
         raise NotImplementedError
+
+    @staticmethod
+    def is_valid_email_list(obj):
+        """Ensures that `obj` (a list of strings) is a valid list of emails"""
+        def is_valid_email(email):
+            return '@' in email and not email.startswith('@') and \
```

Comment:
Do not end lines with backslashes. Please see the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide). Consider formatting as:

     return (
         '@' in email and not email.startswith('@') and
         not email.endswith('@'))

Reply:
done